lib/facter/kmod.rb

```diff
@@ -24,10 +24,8 @@
 
               kmod[directory]['parameters'][param] = File.read("/sys/module/#{directory}/parameters/#{param}").chomp
```

Comment:
Should we not catch exceptions around this single line?

I feel like if` /sys/module/#{directory}/parameters` contains multiple files, and reading one fail, we stop processing the directory and the fact will only have a partial view of the parameters.  Also I am not sure the subset of parameters we get will be the same all the time.

Reply:
Good call, I've just reworked the rescue bits, do they look better now?

Reply:
LGTM!